GrandCentralBoard/GoogleAPIDataProvider.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GoogleAPIDataProvider.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 06.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Alamofire
+
+enum GoogleAPIDataError: ErrorType {
```

Comment:
Sure, but even Apple uses the same format (first one) for inheritance (https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Inheritance.html), and I think also Swiftlint would argue if we would add it to the project.